plugin.cpp

```diff
@@ -1,3 +1,9 @@
+/*
+ * Copyright (C) 2016, Pelagicore AB.
+ * Author: Jonatan Pålsson <jonatan.palsson@pelagicore.com>
+ * SPDX-License-Identify: MPL-2.0
+ */
```

Comment:
This doesn't really look like the template example we have does it? https://github.com/Pelagicore/OpenSourceTemplates/blob/master/examples/Example_header_cpp_mpl.md